data/tools/callGraph.yml

```diff
@@ -0,0 +1,33 @@
+name: callGraph
+categories:
+  - linter
+tags:
+  - awk
+  - bash
+  - basic
+  - dart
+  - fortran
+  - go
+  - lua
+  - javascript
+  - julia
+  - kotlin
+  - matlab
+  - perl
+  - pascal
+  - php
+  - python
+  - R
+  - raku
+  - ruby
+  - rust
+  - scala
+  - swift
+  - tcl
+  - typescript
+license: GNU General Public License
+types:
+  - cli
+source: 'https://github.com/koknat/callGraph'
+homepage: 'https://github.com/koknat/callGraph'
+description: open-source call graph generator for awk, bash, basic, dart, fortran, go, lua, javascript, juila, kotlin, matlab, perl, pascal, php, python, R, raku, ruby, rust, scala, swift, tcl, and typescript
```

Comment:
The tags are already listed above. Could you remove them and write a unique description that describes the advantages of the tool please?